common-app-lib/src/cmr/common_app/services/search/parameter_validation.clj

```diff
@@ -159,6 +159,7 @@
 (def and-option #{:and})
 (def and-or-option #{:and :or})
 (def string-plus-and-options #{:pattern :ignore-case :and})
+(def string-plus-and-exclude-options #{:exclude-collection :pattern :ignore-case :and})
```

Comment:
test lines were added to specific tests, they have comments explaining why they were added which include the ticket name.  It is a little hard to find because of all the whitespace changes in that file.